src/Util.js

```diff
@@ -113,7 +113,7 @@ class Util {
             const reason = (err.response !== undefined && err.response.data !== undefined) ?
                 err.response.data :
                 "UNKNOWN";
-            err.message = method.name + ': ' + err.message + '. Reason: ' + reason;
+            err.info = method.name + ': ' + err.message + '. Reason: ' + reason;
```

Comment:
this obscures the error on safari because Safari throws a type error that assigning to `message` is not allowed because it's read-only instead of the actual error which is IndexedDB related.

Reply:
Should we take out the message from here then since it will be in the error anyway?